src/login/Login.js

```diff
@@ -41,6 +41,9 @@ export default class Login extends React.Component {
         <LoginRow
           label='Password'
           input={<input name='password' type='password' />} />
+        <LoginRow
```

Comment:
Is this text field always displayed? That may be confusing for users who do not need to enter an OTP. Normally OTP is done in a multi-step process and only displayed if the user has OTP enabled. To keep this PR simple, maybe the OTP field could need to be unfolded or at least at a text above it "if you use OTP, enter it".